src/bvals/boundary_conditions.cpp

```diff
@@ -12,9 +12,9 @@
 //========================================================================================
```

Comment:
This file has essentially been completely re-written, so the diff may not be as informative as just looking at the file.